hal/network/ncp_client/quectel/quectel_ncp_client.cpp

```diff
@@ -1793,14 +1809,21 @@ int QuectelNcpClient::processEventsImpl() {
     }
     SCOPE_GUARD({ regCheckTime_ = millis(); });
 
-    // Check GPRS, LET, NB-IOT network registration status
-    if (ncpId() != PLATFORM_NCP_QUECTEL_BG95_M1) {
-        CHECK_PARSER(parser_.execCommand("AT+CEER"));  // Seems to stall the modem on BG95-MF
-        CHECK_PARSER(parser_.execCommand("AT+CMEE?")); // Seems to stall the modem on BG95-MF
```

Comment:
Let's leave this one in as well. It's another error code that is nice to get on supported modems.

Reply:
AT+CMEE? ?  That's just the level of CMEE set which is always 1.